XGDB/jobs/apps_exec.php

```diff
@@ -71,9 +71,15 @@
     }
     $add_app="INSERT INTO $global_DB.apps (app_id, program, platform, nodes, proc_per_node, date_added, description, developer, is_default, max_job_time) values ('$app_id', '$program', '$platform', $nodes, $proc_per_node, now(), '$description', '$developer', '$is_default', '$max_job_time')"; // ADMIN hard-coded on purpose
     $insert=mysql_query($add_app);
-    if($insert)
+    
+    if($insert) # 2-17-16 added this loop which will always retrieve the latest uid (for highlighting) even if records were previously deleted.
     {
-        header("location:apps.php#${next_uid}");
+        $max_query="SELECT MAX(uid) as max_uid from ${global_DB}.apps";

```

Comment:
The indentation of lines 77-81 is really confusing and obfuscates the actual flow of the program. Ideally all of these lines should be aligned, except for 80 which should be indented one additional level.

Little things like this go a long way when someone has to come back a few months later and try to figure out what some piece of code is doing.